src/chrome/content/rules/Steam.xml

```diff
@@ -14,40 +14,65 @@
 		<test url="http://api.steamcommunity.com/" />
 		<test url="http://cdn.steamcommunity.com/public/css/skin_1/header.css" />
 
+	<target host="steamgames.com" />
+		<test url="http://steamgames.com/" />
+	<target host="www.steamgames.com" />
+		<test url="http://www.steamgames.com/" />
 	<target host="partner.steamgames.com" />
+		<test url="http://partner.steamgames.com/" />
 
+	<target host="steampowered.com" />
+		<test url="http://steampowered.com/" />
 	<target host="api.steampowered.com" />
 		<test url="http://api.steampowered.com/ISteamWebAPIUtil/GetSupportedAPIList/v1/" />
-	<target host="kgs.steampowered.com" />
 	<target host="partner.steampowered.com" />
+		<test url="http://partner.steampowered.com/" />
 	<target host="storefront.steampowered.com" />
 		<test url="http://storefront.steampowered.com/app/440/" />
 	<target host="store.steampowered.com" />
 		<test url="http://store.steampowered.com/app/440/" />
 	<target host="support.steampowered.com" />
+		<test url="http://support.steampowered.com/" />
 	<target host="help.steampowered.com" />
+		<test url="http://help.steampowered.com/" />
```

Comment:
This already exists as an implicit `test url`. Please remove all similar test urls.

You can read more here: https://github.com/EFForg/https-everywhere/blob/master/ruleset-testing.md

@cschanaj, didn't the recent change in travis tests prevent this?

Reply:
@Bisaloo #14660 hasn't been merged yet